blocks/footer/footer.css

```diff
@@ -248,9 +255,11 @@ html[dir="rtl"] .ft-main-item:not(:last-child) {
 
 @media (width <= 1024px) {
   .ft-social {
-    grid-template-columns: 1fr 2fr;
-    grid-template-rows: 1fr;
-    grid-gap: 0;
+    /* grid-template-columns: 1fr 2fr; */
+    
+    /* grid-template-rows: 1fr; */
+
+    /* grid-gap: 0; */
```

Comment:
these comments can be removed. Since it is all part of version control, we can revert or refer back the PR later if you need them again.